src/kOS.Safe/Execution/CPU.cs

```diff
@@ -614,8 +615,19 @@ private Variable GetVariable(string identifier, bool barewordOkay = false, bool
             }
             if (failOkay)
                 return null;
-            else
-                throw new KOSUndefinedIdentifierException(identifier.TrimStart('$'), "");
+            // In the case where we were looking for a function pointer but didn't find one, and would
+            // have failed with exception, then it's still acceptable to find a hit that isn't a function
+            // pointer (has no trailing asterisk '*') but only if it's a delegate of some sort:
+            if (identifier.EndsWith("*"))
+            {
+                string trimmedTail = identifier.TrimEnd('*');
+                Variable retryVal = GetVariable(trimmedTail, barewordOkay, failOkay);
```

Comment:
both barewordOkay and failOkay will always be false at this point because we test them both for true and return if they are.

Just raising a flag on the off chance you didnt mean for this to happen.